### PR TITLE
Don't show "Get Started" message when project has replication controllers

### LINKED ISSUE #3146: Get Started message can be shown on project overview with content
openshift_web_console

### PR DESCRIPTION
**Pull request by spadgett**
Fixes #3146

### CODE REVIEW

**Comment by spadgett** [comment]
If you have replication controllers with no current replicas, you'll see "No services to show" rather than "Get Started." I think it's reasonable since the project isn't really empty.

File: assets/app/scripts/controllers/overview.js [hunk 1]
```diff
@@ -354,6 +354,7 @@ angular.module('openshiftConsole')
       var projectEmpty =
         hashSizeFilter($scope.unfilteredServices) === 0 &&
         hashSizeFilter($scope.monopodsByService['']) === 0 &&
```
Comment (liggitt) [inline assets/app/scripts/controllers/overview.js]: check `pods` instead

File: assets/app/scripts/controllers/overview.js [hunk 2]
```diff
@@ -354,6 +354,7 @@ angular.module('openshiftConsole')
...
+        hashSizeFilter($scope.deployments) === 0 &&
         hashSizeFilter($scope.deploymentConfigsByService['']) === 0;
```
Comment (liggitt) [inline assets/app/scripts/controllers/overview.js]: check `deploymentConfigs` instead

**Comment by liggitt** [comment]
LGTM, @smarterclayton merge?

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**